src/containers/CookieBannerContainer/CookieBannerContainer.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react'
+import autobind from 'autobind-decorator'
+import CookieBanner from 'components/CookieBanner'
+import { THIRD_PARTY_INTEGRATIONS } from 'utils/analytics'
+import { getFeatureConfig } from 'utils/features'
+
+const { options } = getFeatureConfig('cookieBanner')
+const ThirdPartyIntegrations = options.map(integration => ({
+  ...integration,
+  initFunc: THIRD_PARTY_INTEGRATIONS[integration.thirdParty],
```

Comment:
It might be a good idea to ensure this doesn't crash if somebody passes an unknown thirdparty integration, when people run their own interface. `console.warn('unknown integration ${intergration.thirdParty}'` or something. Wdyt?

Reply:
@andre-meyer initFunc is market as required in CookieBanner's propTypes, so if its undefined there'll be an error message